defaults/main.yml

```diff
@@ -3,6 +3,7 @@
 he_install_packages:                        false
 he_initial_clean:                           false
 he_pre_checks:                              false
+he_pre_checks_network:                      false
```

Comment:
This is a specific validation and not a stage. I would add a different section like this:
```
# specific validation checks
he_network_interface_check: false
```
If you need more specific checks add them there